app/views/layouts/body/_title.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container site-title-container">
+  <span class="site-title-container01"></span>
```

Comment:
extra spans are there in order to make background colors (different on each side) stretch across all the way